content/Consumer Privacy Regulations CCPA and GDPR - Pablo Blanco.md

```diff
@@ -0,0 +1,240 @@
+“maintains patient information in the same manner” as PHI under HIPAA.
+
+Healthcare providers and insurance agents may assume that the law’s HIPAA
+exemption will cover PHI and other customer information. So we have good news,
+If your business is HIPAA compliant, you are not required to apply extra
+measures to protect your patient’s and customers’ information. Now let’s take a
+look at some examples where the exemption applies and not.
+
+**Case 1**: Assume that you own a sportswear company that sells products in
+California and you have developed a step-counter app that users can download to
+their phones through Google Play and App Store. The app tracks the number of
+steps a user takes each day and captures additional information, including the
+user's name, birthday, weight, calories burned, geolocation, and walking
+average. That company is probably not a covered entity or business associate
+under HIPAA and would not be able to apply to the CCPA's HIPAA exemption.
+
+**Case 2**: Now consider a health care system "operating" in California that
+created an app with the same features as the previous case, yet made the app
+available only to its patients to monitor their health and treat medical
+conditions. That organization is a covered entity under HIPAA, the data is
+probably PHI, and the HIPAA exemption probably applies.
+
+## Final thoughts
+
+Data privacy regulations will demand a lot of work in the preparedness to be
+compliance, but being forced to optimizing the data that a company collects,
+processes and stores could create efficiency. Companies need only store the
+necessary pieces of personal information to perform their services and reduce
+the time and resources (they have spent in the past) to store all of it.
+Compliance can be viewed as a company’s competitive advantage by offering
+consumers peace of mind that their data is protected and secure.
+
+I believe that CCPA is a powerful law that will be a big win for both consumers
+and businesses, and will help to put the United States at the same level as
+Europe with GDPR in the global privacy discussion. This law came to stay and
+leverage the rights between consumers and business, giving the former the chance
+to know what a company knows about them, decide whether or not they can keep
+that information, and prevent them from selling it.
+
+Governments are beginning to take data privacy very seriously. Like in the
+European Union, Brazil, India, Japan, and other countries, the CCPA will have
+far more impacts across state jurisdictions, reaching companies around the
+world. Getting prepared to be compliant is important and will save your company
+a lot of money, time, and reputation. To be compliant with these regulations can
+be a hard work to achieve, you need to be able to identify content related to a
+data subject, classify and protect consumer data, and sometimes even delete upon
+request. But there is some light at the end of the tunnel, if you get your
```

Comment:
This last paragraph feels somewhat redundant, how about sth like:
'The upside of this effort is gaining a competitive advantage, both in the present and looking ahead at future regulations on the horizon'.